backend/build.gradle.kts

```diff
@@ -58,7 +58,7 @@ tasks {
 
     val bootJar by getting(BootJar::class) {
         archiveName = "globe42.jar"
-        dependsOn(":frontend:npm_run_build")
+        dependsOn(":frontend:assemble")
```

Comment:
For my curiosity: what is the point of this?

Reply:
assemble is a standard lifecycle task of the base plugin (https://docs.gradle.org/current/userguide/base_plugin.html) which is supposed to... assemble the artifact of the project. But it didn't do anything before, which is a bit strange.